optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/KieRuntimeBuilderWrapper.java

```diff
@@ -17,7 +17,7 @@
 package org.optaplanner.core.impl.score.director.drools;
 
 import org.kie.api.KieBase;
-import org.kie.kogito.rules.KieRuntimeBuilder;
+import org.kie.kogito.legacy.rules.KieRuntimeBuilder;
```

Comment:
How can we use the new API instead of the legacy one?

Reply:
Using rule units.